docs/pages/guides/using-sentry.mdx

```diff
@@ -16,20 +18,199 @@ import { CODE } from '~/ui/components/Text';
+
+```js
+import * as Sentry from '@sentry/react-native';
+
+Sentry.init({
+  dsn: 'YOUR DSN HERE',
+  debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
+});
+```
+
+Now wrap the root component of your app with Sentry. This may be **App.js** or **index.js** depending on your project, or in [**app/\_layout.js** with Expo Router](https://docs.expo.dev/router/layouts/#create-a-layout-route).
+
+```js
+import * as Sentry from '@sentry/react-native';
+
+// Your App component here
+
+export default Sentry.wrap(App);
+```
+
+</Step>
+
+<Step label="6">
+
+### Verify the configuration
+
+Create a new release build of your app and verify that it uploads source maps correctly. You may want to add a button in your app to test that it is working and sourcemaps are wired up as expected, for example:
+
+```jsx
+import { Button } from 'react-native';
+
+// Inside some component
+<Button title="Press me" onPress={() => throw new Error('Hello, Sentry!')} />;
+```
+
+</Step>
+
+## Usage with EAS Build
+
+Ensure that `SENTRY_AUTH_TOKEN` is set in your build environment, and Sentry will automatically upload source maps for you. If you use environment variables rather than properties in your app config, ensure that those are set as well.
+
+Using the above instructions, no additional work is needed to integrate Sentry into your project when using EAS Build.
+
+## Usage with EAS Update
+
+### Tag configuration
+
+Configuring Sentry to tag your scope with information about your update allows to see errors happening on certain updates in Sentry dashboard.
+
+Add the following snippet in the global scope as early as possible in your application's lifecycle.
+
+```js
+import * as Updates from 'expo-updates';
+
+const { metadata, extra } = Updates.manifest;
+
+Sentry.init({
+  // dsn, release, dist, etc...
+});
+
+Sentry.configureScope(scope => {
+  scope.setTag('expo-update-group-id', metadata?.updateGroup);
+  scope.setTag('expo-update-id', Updates.updateId);
+  // This will be `true` if the update is the one embedded in the build, and not one downloaded from the updates server.
+  scope.setTag('expo-is-embedded-update', Updates.isEmbeddedLaunch);
+  scope.setTag(
+    'expo-update-debug-url',
+    Updates.isEmbeddedLaunch
+      ? 'not applicable for embedded updates'
+      : `https://expo.dev/accounts/${extra?.expoClient?.owner}/projects/${extra?.expoClient?.slug}/updates/${metadata?.updateGroup}`
+  );
+});
+```
+
+Once configured, information about the associated update will show up in an error's tag section:
+
+<ImageSpotlight alt="Sentry update tags" src="/static/images/guides/sentry-tags.png" />
+
+### Upload source maps
+
+After running `eas update`, upload the source maps to Sentry:
+
+```sh
+SENTRY_AUTH_TOKEN=[super-secret-token] node node_modules/@sentry/react-native/scripts/expo-upload-sourcemaps.js dist
```

Comment:
Let's use the `npx` command so we are not bound to the script path in the future. 

Plus it's shorter.

```suggestion
SENTRY_AUTH_TOKEN=[super-secret-token] npx sentry-expo-upload-sourcemaps dist
```